lib/schema_web/controllers/schema_controller.ex

```diff
@@ -122,6 +135,45 @@ defmodule SchemaWeb.SchemaController do
     send_json_resp(conn, version)
   end
 
+  @doc """
+  Get available OCSF schema versions.
+  """
+  swagger_path :versions do
+    get("/api/versions")
+    summary("Versions")
+    description("Get available OCSF schema versions.")
+    produces("application/json")
+    tag("Schema")
+    response(200, "Success", :Versions)
+  end
+
+  @spec versions(Plug.Conn.t(), any) :: Plug.Conn.t()
+  def versions(conn, _params) do
+
+    url = Application.get_env(:schema_server, SchemaWeb.Endpoint)[:url]
+
+    base_url = "#{conn.scheme}://#{Keyword.fetch!(url, :host)}:#{Keyword.fetch!(url, :port)}"
+
+    available_versions = Schemas.versions()
+    |> Enum.map(fn {version, _} -> version end)
+
+    default_version = %{:version => Schema.version(), :url => "#{base_url}/api"}
+
+    versions_response = case available_versions do
+      [] ->
+        # If there is no response, we only provide a single schema
+        %{:versions => [default_version], :default => default_version}
```

Comment:
```suggestion
        %{:versions => ["#{base_url}/#{Schema.version()}/api"], :default => default_version}
```
Put version in `versions` section for empty list case, making it consistent with non-empty case.

Reply:
As far as I can tell, the current state is already consistent with the non-empty case.

Versions is supposed to be a list of maps. `default_version` is the map for the default version.

Reply:
I just figured the `versions` attribute URLs should always include the version.

Reply:
okay, but in this case we can adapt `default_version` to change.

Reply:
```
{"default":{"version":"1.0.0-rc.3","url":"http://localhost:8080/1.0.0-rc.3/api"},"versions":[{"version":"1.0.0-rc.3","url":"http://localhost:8080/1.0.0-rc.3/api"},{"version":"1.1.0-dev","url":"http://localhost:8080/1.1.0-dev/api"}]}
```

This is what will be returned then